app/uk/gov/hmrc/economiccrimelevyregistration/views/FirstContactNameView.scala.html

```diff
@@ -41,7 +39,7 @@ <h1 class="govuk-heading-l">@messages("firstContactName.heading")</h1>
             @govukInput(
                 InputViewModel(
                     field = form("value"),
-                    label = LabelViewModel(HtmlContent(labelHtml))
+                    label = LabelViewModel(Text(""))
```

Comment:
We should still be setting this label rather than leaving it empty

I wonder if wrapping the `h1` in a `<legend>` tag would resolve the issue, e.g:

```<legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
                   <h1 class="govuk-fieldset__heading">@messages("firstContactName.heading")</h1>
              </legend>
```

Ideally - whatever solution we come up with it would be great if we could test without merging first 🤔

Reply:
Better if we keep the following issues separate:

* ISSUE 1
  Let software developers produce the accessiblity report on the local environment **before** having to merge changes into the main development line

* ISSUE 2
  Agree on the right HTML structure able to display header and paragraph along with forms (it may be the adoption of the `legend` element or something else)
  

Fixing issue number 1 requires a good knowledge of the HMRC continuous build robots (currently implemented using Jenkins pipelines). I really hope James Varghese can help.

Fixing issue number 2 with the adoption of the `legend` element seems to translate in more Scala/HTML code and larger effort. I'm unsure the effort would be worth as it could not pass the accessiblity test (see https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/)

Reply:
@peter-hazell ,

I tried hard with your proposed solution (the one suggesting to wrap `h1` into a `legend` element). Changes are committed here (on a separate `ECL-252-using-legend` branch):

https://github.com/hmrc/economic-crime-levy-registration-frontend/blob/ECL-252-using-legend/app/uk/gov/hmrc/economiccrimelevyregistration/views/FirstContactNameView.scala.html#L29-L35

I am sorry the accessibility report complains as per attached screenshot:

<img width="1188" alt="when_legend_wraps_h1_and_p" src="https://user-images.githubusercontent.com/85297193/219965263-25029ed1-e1b6-4218-8d66-5f5e8dc070bf.png">

Therefore, I'm continuing with my initial proposed solution ... but making sure labels are **not** set to empty text.

Reply:
I added detailed instructions on how to produce accessibility reports locally. My instructions are customised for this registration frontend service .

It's commit hash 44f6a8357a

@peter-hazell please advice if you'd prefer those instructions to be written somewhere else (or not written at all).

Reply:
Finally, thanks to commit hash `925ac28ef` (included in this pull request) the accessiblity issues counter is going down to zero! 

See attached final report:

<img width="1176" alt="successful_report" src="https://user-images.githubusercontent.com/85297193/219965750-2167e684-d857-4462-bda2-e6270ea36a73.png">

Reply:
Thanks @pangiole-hmrc 

Whilst this solution does solve the accessibility issues - I don't think this is necessarily the right solution, a couple of things:

- The new labels you have set display on the UI - which we do not want, for example 'AML supervisor' shouldn't be there
![image](https://user-images.githubusercontent.com/18005888/220058461-d322c134-6cd1-4669-bd4a-e74b38a8426a.png)

-- I think a nicer possible solution will be to just set the piece of text for the form input as the label, and move any other paragraphs or h1 outside of the label - I believe this issue is happening because we have set a h1 and other pieces of text within the same block - but the label needs to just be one piece of text

Reply:
**Update:**

- Had a catch up within the team and we have reached the conclusion, see commit https://github.com/hmrc/economic-crime-levy-registration-frontend/pull/72/commits/140d5e56e90fae2abb70cca6a462c312af03bac2 for example
- We should be setting the legend/label as the page heading but visually hidden, as per discussion here https://hmrcdigital.slack.com/archives/CJMUM9AG3/p1676884809660379 and also guidance here https://design-system.service.gov.uk/get-started/labels-legends-headings/
- Accessibility test instructions to be moved to the acceptance tests repo
- Regarding the error 'Element “option” without attribute “label” must not be empty.' - this is due to the fact we have to set an empty option for the accessible autocomplete, so we either need to come up with a solution that means the accessible autocomplete continues to work, or ignore this error